.travis.yml

```diff
@@ -109,6 +109,17 @@ install:
       ${SUCCESS}
     ;;
     esac
+  # Helper function to work around Travis timeouts based on lack of output
```

Comment:
Maybe a comment (as per your commit description) explaining why travis_wait is not used. Quite unfortunate you have to do something like this.

Reply:
I have updated the comment to list reasons why this custom wrapper is used instead of `travis_wait`.

And by the way, if you prefer to use `travis_wait` instead that is fine. I don't have a strong preference either way, just implemented what seemed like a better solution to me.